forkmanager/fork.go

```diff
@@ -18,21 +18,33 @@ type Fork struct {
 	Name string
 	// after the fork is activated, `FromBlockNumber` shows from which block is enabled
 	FromBlockNumber uint64
+	Params          *chain.ForkParams
```

Comment:
Nitpick: add some comment, even though it may be obvious what params stand for.

E.g.
```suggestion
    // fork consensus parameters
	Params          *chain.ForkParams
```